staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator/aggregator.go

```diff
@@ -189,6 +189,7 @@ func (s *specAggregator) buildOpenAPISpec() (specToReturn *spec.Swagger, err err
 			specToReturn.Paths = nil
 			specToReturn.Definitions = nil
 		}
+		specInfo.spec.Definitions = PruneDefaults(specInfo.spec.Definitions)
```

Comment:
Aggregated API servers won't call this. Hence, they will expose the defaults. But as this goes through the aggregrator before the clients see it, it's probably ok.

Reply:
Thanks, I think that's the reason for the current test failure :-)